src/components/Main/MainPageContent.tsx

```diff
@@ -1,35 +1,36 @@
-import ContentList from './ContentList';
-import { Spinner, Box, Text, Flex } from '@chakra-ui/react';
-import useObserver from '../hooks/useObserver';
-import { useState, useRef, useCallback } from 'react';
+import useObserver from '../../hooks/useObserver';
 import axios from 'axios';
+import ContentList from '../ContentList';
+import { useState, useRef, useCallback } from 'react';
+import { Spinner, Box, Text, Flex, useBoolean } from '@chakra-ui/react';
 
 const axiosMainData = async (offset = 0, limit = 0) => {
   return await axios.get(
     // 정호님이 만드신 요청으로 변경이 필요할 수 있습니다
-    `https://kdt.frontend.4th.programmers.co.kr:5009/posts/channel/64f806ccb3b4d210bb7b4fcb?offset=${offset}&limit=${limit}`
+    `https://kdt.frontend.4th.programmers.co.kr:5009/posts/channel/64fbfdabe8468d1ed414e307?offset=${offset}&limit=${limit}`
   );
 };
 
-type contents = [] | { title: string; _id: string }[];
+export type Content = {
+  title: string;
+  _id: string;
+  image?: string;
+};
 
 function MainPageContent() {
+  const [contents, setContents] = useState<Content[]>([]);
+  const [isContentEmpty, setIsContentEmpty] = useBoolean();
+  const contentCount = useRef<number>(0);
```

Comment:
```suggestion
  const contentCount = useRef(0);
```
타입 추론이 되어서 제거하셔도 될 것 같아요!!